examples/images_contours_and_fields/interpolation_methods.py

```diff
@@ -26,7 +27,7 @@
 fig.subplots_adjust(hspace=0.3, wspace=0.05)
 
 for ax, interp_method in zip(axes.flat, methods):
-    ax.imshow(grid, interpolation=interp_method)
+    ax.imshow(grid, interpolation=interp_method, cmap='viridis')
     ax.set_title(interp_method)
 
-plt.show()
+plt.savefig("test.png")
```

Comment:
no need to change this. These are demos, so don't we want it to show() so that the image ends up in the documentation?